README.md

```diff
@@ -1,113 +1,96 @@
+## Lower-Level Details (For Experienced Istio Users)
 
-### Issuer or ClusterIssuer
-An Issuer or ClusterIssuer must be configured which will be used to sign Istio
-certificates against.
+⚠️  The [getting started](./docs/getting_started.md) guide is a better place if you just want to try istio-csr out!
 
-Here are examples of CA [Issuer](./docs/example-issuer.yaml) and
-[ClusterIssuer](./docs/example-cluster-issuer.yaml) configurations that are
-bootstrapped through self-signed issuers. It is advised to not use the CA Issuer
-type in production environments, and instead use an issuer who's CA private key
-material does not reside within the cluster.
+Running istio-csr requires a few steps and preconditions in order:
 
-> It is important to use an issuer type that is able to sign Istio mTLS workload
-> certificates (SPIFFE URI SANs) and
-> [istiod serving certificates](./deploy/charts/istio-csr/templates/certificate.yaml).
-> ACME issuers will not work.
+1. A cluster _without_ Istio already installed
+2. cert-manager [installed](https://cert-manager.io/docs/installation/) in the cluster
+3. An `Issuer` or `ClusterIssuer` which will be used to issue Istio certificates
+4. istio-csr installed (likely via helm)
+5. Istio [installed](https://istio.io/latest/docs/setup/install/istioctl/) with some required custom config, e.g. in `./hack/istio-config-1.10.0.yaml`
 
-If using an Issuer rather than the ClusterIssuer type, the Issuer must reside in
-the same namespace as that configured by `--certificate-namespace` on istio-csr,
-`istio-system` by default.
+### Why Custom Istio Install Manifests?
 
-### Installing istio-csr
+If you take a look at the contents of [`istio-config-1.10.0.yaml`](./hack/istio-config-1.10.0.yaml) there are a few
+custom configuration options which are important.
 
-Next, install istio-csr into the cluster, configured to use the Issuer or
-ClusterIssuer installed earlier.
+Required changes include setting `ENABLE_CA_SERVER` to `false` and setting the `caAddress` from which Istio will
+request certificates; replacing the CA server is the whole point of istio-csr!
 
-> ⚠️ It is highly recommended that the root CA certificates are statically
-> defined in istio-csr. If they are not, istio-csr will "discover" the root CA
-> certificates when requesting its serving certificate. Although discovering the
-> root CAs reduces operational complexity, using CA pinning with a static bundle
-> is less vulnerable to
-> [signer hijacking attacks](https://github.com/cert-manager/istio-csr/issues/103#issuecomment-923882792),
-> for example if a signer's token (such as cert-manager-controller) was stolen.
+Mounting and statically specifying the root CA is also an important recommended step. Without a manually specified
+root CA istio-csr defaults to trying to discover root CAs automatically, which could theoretically lead to a
+[signer hijacking attack](https://github.com/cert-manager/istio-csr/issues/103#issuecomment-923882792).
```

Comment:
This line is missing `for example if a signer's token (such as cert-manager-controller) was stolen.`

Reply:
Yeah good addition, I'll do that :+1: